### PR TITLE
import UWP content from IntelliSense - Windows.UI.Xaml.Media.*

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: xml/Windows.UI.Xaml.Media.Animation/RepeatBehavior.xml [hunk 1]
```diff
@@ -67,9 +69,11 @@
         <Parameter Name="duration" Type="System.TimeSpan" />
       </Parameters>
       <Docs>
-        <param name="duration">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="duration">The total length of time that the Windows.UI.Xaml.Media.Animation.Timeline should play (its active duration).</param>
+        <summary>Initializes a new instance of the <see cref="T:Windows.UI.Xaml.Media.Animation.RepeatBehavior" /> struct with the specified repeat duration.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentOutOfRangeException">
+          <paramref name="duration" /> evaluates to a negative number.</exception>
```
Comment (rpetrusha) [inline xml/Windows.UI.Xaml.Media.Animation/RepeatBehavior.xml]:
```suggestion
          <paramref name="duration" /> evaluates to a negative time interval.</exception>
```

File: xml/Windows.UI.Xaml.Media.Animation/RepeatBehavior.xml [hunk 2]
```diff
@@ -378,10 +384,10 @@
         <Parameter Name="formatProvider" Type="System.IFormatProvider" />
       </Parameters>
       <Docs>
-        <param name="format">To be added.</param>
-        <param name="formatProvider">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="format">The string specifying the format to use, or <see langword="null" /> to use the default format defined for the type of the <see cref="T:System.IFormattable" /> implementation.</param>
+        <param name="formatProvider">The <see cref="T:System.IFormatProvider" /> to use to format the value, or <see langword="null" /> to obtain the numeric format information from the current locale setting of the operating system.</param>
```
Comment (rpetrusha) [inline xml/Windows.UI.Xaml.Media.Animation/RepeatBehavior.xml]:
```suggestion
        <param name="formatProvider">An object that provides formatting information for value, or <see langword="null" /> to obtain the numeric format information from the current culture.</param>
```

File: xml/Windows.UI.Xaml.Media.Media3D/Matrix3D.xml [hunk 3]
```diff
@@ -105,9 +105,9 @@
         <Parameter Name="o" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="o">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="o">The object to test for equality.</param>
+        <summary>Tests equality between two matrices.</summary>
```
Comment (rpetrusha) [inline xml/Windows.UI.Xaml.Media.Media3D/Matrix3D.xml]:
```suggestion
        <summary>Tests for equality between two matrices.</summary>
```

File: xml/Windows.UI.Xaml.Media.Media3D/Matrix3D.xml [hunk 4]
```diff
@@ -105,9 +105,9 @@
         <Parameter Name="o" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="o">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="o">The object to test for equality.</param>
```
Comment (rpetrusha) [inline xml/Windows.UI.Xaml.Media.Media3D/Matrix3D.xml]:
```suggestion
        <param name="o">The object to test for equality with the current instance.</param>
```

File: xml/Windows.UI.Xaml.Media.Media3D/Matrix3D.xml [hunk 5]
```diff
@@ -131,9 +131,9 @@
         <Parameter Name="value" Type="Windows.UI.Xaml.Media.Media3D.Matrix3D" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The <see cref="T:Windows.UI.Xaml.Media.Media3D.Matrix3D" /> to compare to.</param>
+        <summary>Tests equality between two matrices.</summary>
```
Comment (rpetrusha) [inline xml/Windows.UI.Xaml.Media.Media3D/Matrix3D.xml]:
```suggestion
        <summary>Tests for equality between two matrices.</summary>
```

File: xml/Windows.UI.Xaml.Media.Media3D/Matrix3D.xml [hunk 6]
```diff
@@ -160,8 +160,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Returns the hash code for this matrix.</summary>
+        <returns>An integer that specifies the hash code for this matrix.</returns>
```
Comment (rpetrusha) [inline xml/Windows.UI.Xaml.Media.Media3D/Matrix3D.xml]:
```suggestion
        <returns>The hash code for this matrix.</returns>
```

File: xml/Windows.UI.Xaml.Media.Media3D/Matrix3D.xml [hunk 7]
```diff
@@ -626,10 +627,10 @@
         <Parameter Name="matrix2" Type="Windows.UI.Xaml.Media.Media3D.Matrix3D" />
       </Parameters>
       <Docs>
-        <param name="matrix1">To be added.</param>
-        <param name="matrix2">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="matrix1">The first matrix to compare.</param>
+        <param name="matrix2">The second matrix to compare.</param>
+        <summary>Compares two <see cref="T:Windows.UI.Xaml.Media.Media3D.Matrix3D" /> instances for exact equality.</summary>
```
Comment (rpetrusha) [inline xml/Windows.UI.Xaml.Media.Media3D/Matrix3D.xml]:
```suggestion
        <summary>Compares two <see cref="T:Windows.UI.Xaml.Media.Media3D.Matrix3D" /> instances for equality.</summary>
```

File: xml/Windows.UI.Xaml.Media.Media3D/Matrix3D.xml [hunk 8]
```diff
@@ -717,10 +718,10 @@
         <Parameter Name="provider" Type="System.IFormatProvider" />
       </Parameters>
       <Docs>
-        <param name="format">To be added.</param>
-        <param name="provider">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="format">The format to use.</param>
+        <param name="provider">The provider to use.</param>
+        <summary>For a description of this member, see <see cref="M:System.IFormattable.ToString" />.</summary>
```
Comment (rpetrusha) [inline xml/Windows.UI.Xaml.Media.Media3D/Matrix3D.xml]:
```suggestion
        <summary>For a description of this member, see <see cref="M:System.IFormattable.ToString(System.String,System.IFormatProvider)" />.</summary>
```

File: xml/Windows.UI.Xaml.Media.Media3D/Matrix3D.xml [hunk 9]
```diff
@@ -772,9 +773,9 @@
         <Parameter Name="provider" Type="System.IFormatProvider" />
       </Parameters>
       <Docs>
-        <param name="provider">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="provider">Culture-specified formatting information.</param>
```
Comment (rpetrusha) [inline xml/Windows.UI.Xaml.Media.Media3D/Matrix3D.xml]:
```suggestion
        <param name="provider">An object that provides culture-specified formatting information.</param>
```

File: xml/Windows.UI.Xaml.Media/Matrix.xml [hunk 10]
```diff
@@ -405,10 +405,10 @@
         <Parameter Name="provider" Type="System.IFormatProvider" />
       </Parameters>
       <Docs>
-        <param name="format">To be added.</param>
-        <param name="provider">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="format">The string specifying the format to use, or <see langword="null" /> to use the default format defined for the type of the <see cref="T:System.IFormattable" /> implementation.</param>
+        <param name="provider">The <see cref="T:System.IFormatProvider" /> to use to format the value, or <see langword="null" /> to obtain the numeric format information from the current locale setting of the operating system.</param>
```
Comment (rpetrusha) [inline xml/Windows.UI.Xaml.Media/Matrix.xml]:
```suggestion
        <param name="provider">An object that provides formatting information, or <see langword="null" /> to obtain the numeric format information from the current culture.</param>
```

File: xml/Windows.UI.Xaml.Media.Animation/RepeatBehavior.xml [hunk 11]
```diff
@@ -24,7 +24,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Describes how a Windows.UI.Xaml.Media.Animation.Timeline repeats its simple duration.</summary>
```
[the remark this answers is not in the document]
Comment (mairaw) [reply xml/Windows.UI.Xaml.Media.Animation/RepeatBehavior.xml]: It's uwp: https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Media.Animation.Timeline

Comment (mairaw) [inline xml/Windows.UI.Xaml.Media.Animation/RepeatBehavior.xml]:
```suggestion
    <summary>Describes how a <see cref="T:Windows.UI.Xaml.Media.Animation.Timeline" /> repeats its simple duration.</summary>
```

**Pull request merged**